@here/harp-mapview/lib/text/TextElementState.ts

```diff
@@ -155,6 +155,7 @@ export class TextElementState {
             }
         }
         this.m_viewDistance = undefined;
+        this.element.textBufferObject = undefined;
```

Comment:
This I think is mandatory! How we could keep TextBufferObject even is labels state is completely reset...

Reply:
I suppose we keep it to reuse it if the label becomes visible again.

Reply:
I think this will no longer work with alternative placement. When a label becomes invisible and visible again the placement of the label may change. And we saw some issues that the offset and screen position of a label was not correctly applied when reusing the `textBufferObject`.